code/modules/research/plortmachine.dm

```diff
@@ -0,0 +1,57 @@
+/**********************Plort Redemption Machine**************************/
+//Accepts slime "Cores" as they are definetly NOT called, and turns them into stuff. Probably money? maybe goods? idk im writing this thing before i code it.
+
+/obj/machinery/plortrefinery
+	name = "plort redemption machine"
```

Comment:
plorts are from slime rancher, it should be called an extract redemption machine to match our vernacular

Reply:
more slimerancher = good